api/tests/Api/SnapshotTests/__snapshots__/ResponseSnapshotTest__testOpenApiSpecMatchesSnapshot__1.yml

```diff
@@ -680,11 +693,12 @@ components:
       type: object
     Activity.jsonhal-read_ScheduleEntry.Activity:
       deprecated: false
-      description: 'A piece of programme that will be carried out once or multiple times in a camp.'
+      description: ''
```

Comment:
I find it strange that the descriptions of the derived models are now missing,
but not of all derived models, but when then only of derived models.
For me it is ok if we make the update first and then try to fix the documentation again